logging/cluster-logging-external.adoc

```diff
@@ -43,10 +43,13 @@ include::modules/cluster-logging-collector-log-forward-syslog.adoc[leveloffset=+
 include::modules/cluster-logging-collector-log-forward-cloudwatch.adoc[leveloffset=+1]
 
 include::modules/cluster-logging-collector-log-forward-loki.adoc[leveloffset=+1]
+include::modules/cluser-logging-troubleshooting-loki-entry-out-of-order-messages.adoc[leveloffset=+2]
```

Comment:
It looks like the new .adoc file's filename has a misspelling: "cluser" rather than "cluster"

Reply:
Updated. Thanks.